modules/images-other-jenkins-oauth-auth.adoc

```diff
@@ -18,12 +18,28 @@ Jenkins supports both browser and non-browser access.
 
 Valid users are automatically added to the Jenkins authorization matrix at log
 in, where {product-title} `Roles` dictate the specific Jenkins permissions the
-users have.
+users have.  The `Roles` used by default are the `admin`, `edit`, and `view` 
+`Roles` in the `Project` or namespace that Jenkins is running in.
 
 Users with the `admin` role have the traditional Jenkins administrative
 user permissions. Users with the `edit` or `view` role have progressively
 fewer permissions.
 
+Those defaults noted, the which set of {product-title} `Roles` and which Jenkins permissions
+those `Roles` are assigned in the Jenkins instance are configurable.
+
+The login plugin will look for a `ConfigMap` named `openshift-jenkins-login-plugin-config`. Specifically, when running Jenkins in an {product-title} `Pod`, it will look in the namespace that Jenkins is running in. Otherwise, it looks in the namespace specified in the "client ID" as explained in "Secondary Scenarios" down below.
+
+If this plugin finds and can read in that ConfigMap, it then:
```

Comment:
For the following statements, is the login-plug in looking for `openshift-jenkins-login-plugin-config` in the `client ID`? And what is the plug-in finding and reading? Is it `openshift-jenkins-login-plugin-config` too?: 

"Otherwise, it looks in the namespace specified in the "client ID" as explained in "Secondary Scenarios" down below.

If this plugin finds and can read in that ConfigMap, it then:"

Could it be changed to something like (please forgive me if I'm way off): 

"Otherwise, the login plug-in looks in the namespace specified in the `client ID` as explained in secondary scenarios outlined in the following list.

If the login plug-in finds and can read the `openshift-jenkins-login-plugin-config` `ConfigMap`, you can edit the `Role`s:"

Reply:
Let's delete the secondary scenario piece - should not have included it.  It is touched upon in the README but is not something we want to advertise in the official doc.